backend/src/main/java/net/vidux/camhub/camera/Camera.java

```diff
@@ -1,18 +1,25 @@
-package net.vidux.camhub.models;
+package net.vidux.camhub.camera;
 
 import java.util.Date;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
 
 @Getter
+@NoArgsConstructor
 @AllArgsConstructor
+@Entity
 @Builder
 public class Camera {
-  private final String name;
+  private String name;
   private String ip;
   private String firmware;
   private Date lastSeen;
   private String password;
+  @Id
```

Comment:
Az sn az a Camare business key-e, natural id-ja. Az adatbázisokban, viszont nem használjuk ezeket elsődleges kulcsként, mert nem elég kompaktak, rontanak az adatbázis performance-n. Kapcsolódó cikk: https://vladmihalcea.com/the-best-way-to-map-a-naturalid-business-key-with-jpa-and-hibernate/

Reply:
Igen, azt éreztem, hogy a string nem lesz túl optimális elsődleges kulcs. Hozzáadtam mint natural ID, és fölvettem egy longot... meg a db initet is át kellett akkor már írni builderesre a generatedvalue miatt